.travis.yml

```diff
@@ -1,8 +1,10 @@
 version: ~> 1.0 # enables build config validation beta feature: https://docs.travis-ci.com/user/build-config-validation
 language: node_js
 node_js:
-  - '12.18.4' #TODO: change it to 'node' once Travis supports the newest version
+  - 14
```

Comment:
you've added **14** and **16** and removed previous **12.18.4**. However, it seems though that you are only using 16. I would not add a new version if its not being used specifically. Alternatively, please leave '12.18.4' in there and node 16 can then be an alternative if Travis allows it.